views/layouts/main.handlebars

```diff
@@ -77,6 +82,13 @@
               </div>
               {{/if}}
             </div>
+            <p>
```

Comment:
Did I already delete this element? If you see the latest file, there is no more language displayed in text

Reply:
@wildan3105 Should I remove the `<p> ..... </p>` and send PR again ??

Reply:
Yes, could you compare to the latest file? Or just do `git rebase` to match to the latest one

Reply:
Ook .. :)